packages/application-shell/src/components/application-shell/application-shell.tsx

```diff
@@ -536,7 +539,7 @@ const ApplicationShell = <AdditionalEnvironmentProperties extends {}>(
 
   return (
     <>
-      <ThemeProvider theme="default" />
+      <ThemeProvider theme="default" themeOverrides={themesOverrides.default} />
```

Comment:
We will need to move the location of the `ThemeProvider` down the tree as it would need the `FlopFlipProvider` to be setup so we can read the feature flag we will use to know which theme to use.
No sure if we would also need to move the `GlobalStyles`.